stack/stack_mac.yaml

```diff
@@ -322,24 +323,30 @@ yafyaml:
 mapl:
   build: YES
   repo: GEOS-ESM
-  version: v2.7.3
+  version: v2.12.2
+  cmake_opts: "-DBLA_VENDOR=Apple"
+  STACK_mapl_FFLAGS=" -fallow-argument-mismatch -fallow-invalid-boz"
 
 geos:
   build: YES
   version: 3.8.1
 
+libtiff:
+  build: YES
+  version: 4.0.3
+
 sqlite:
   build: YES
   version: 3.32.3
 
 proj:
-  build: NO
+  build: YES
   version: 7.1.0
   cmake_opts: "-DENABLE_CURL=OFF -DBUILD_PROJSYNC=OFF"
 
 miniconda3:
   build: NO
-  version: 4.7.12
+  version: 4.6.14
```

Comment:
What's the reason for lowering the version?

Reply:
What about the version of miniconda3 here? Why go from 4.7 to 4.6?

Reply:
It could be updated to 4.7.12 ( not building )

Reply:
Nevermind, I this is fine. It's the version that `stack_noaa` uses, and it's not building.